### DISCUSSION TITLE
Multilanguage support?

### DISCUSSION THREAD

**Discussion by user9931** [TriliumNext/Trilium#2407] [General]
Any plans to add multi language support? There's not so many text in the interface to translate but can be convinient to use for some users....

As more speficic question, how to create "Today" note with non-english date?

**Comment by zadam** [comment]
Multi language support is not planned, at least not in near/medium term.

There are few [date pattern](https://github.com/zadam/trilium/wiki/Day-notes#date-pattern) options to configure but it might not be what you're looking for.

What you could do is to modify the script in the today button - let the backend create the day note and then change the title of the note according to your needs. For that you need scripting/programming skills though.
